build.sbt

```diff
@@ -305,8 +303,57 @@ lazy val `play-ahc-ws-standalone` = project
     `play-ws-standalone`
   ).aggregate(
     `shaded`
+  ).disablePlugins(sbtassembly.AssemblyPlugin)
+
+//---------------------------------------------------------------
+// JSON Readables and Writables
+//---------------------------------------------------------------
+
+lazy val `play-ahc-ws-standalone-json` = project
```

Comment:
well the problem here is probably that we need the underlying response/request type, which is a ahc class.